src/ol/interaction/selectinteraction.js

```diff
@@ -68,6 +77,12 @@ ol.interaction.Select = function(opt_options) {
    */
   this.multi_ = goog.isDef(options.multi) ? options.multi : false;
 
+  /**
+   * @private
+   * @type {ol.interaction.SelectFilterFunction}
+   */
+  this.filter_ = goog.isDef(options.filter) ? options.filter : null;
```

Comment:
The new `filter` option needs to be added to `olx.interaction.SelectOptions`. See the `externs/olx.js` file.